app/models/application_choice.rb

```diff
@@ -1,4 +1,6 @@
 class ApplicationChoice < ApplicationRecord
+  before_create :set_vendor_id
```

Comment:
I am not sure about how the team feels about callbacks in the models,
but I think we should avoid them, to avoid spaghetti code (I love 🍝 but in a different context)

Reply:
I think this is fine for setting defaults, I don't think there's much risk in this becoming spaghetti.